test/test-application/src/index.ts

```diff
@@ -15,13 +15,17 @@
  */
 
 'use strict';
-import {EchoClient} from 'showcase-echo-client';
-import {ShowcaseServer} from 'showcase-server';
+import { EchoClient, SequenceServiceClient,protos } from 'showcase-echo-client';
+
+
 
 import * as assert from 'assert';
-import {promises as fsp} from 'fs';
+import { promises as fsp } from 'fs';
 import * as path from 'path';
-import {protobuf, grpc, GoogleError, GoogleAuth} from 'google-gax';
+import { protobuf, grpc, GoogleError, GoogleAuth} from 'google-gax';
+import { google } from 'google-gax/build/protos/operations';
+import { Stream } from 'stream';
+import stream = require('stream');
```

Comment:
These two lines together look weird, what's the reason? Would 

```ts
import stream from 'stream';
const Stream = stream.Stream;
```
be the same?

Reply:
I just got rid of line 27 - I think we actually don't need it and it was leftover from an earlier attempt